### PR TITLE
Reinstall only Ubuntu

### LINKED ISSUE #168: WSL Uninstall Instructions
The current linked resource for uninstalling (and then reinstalling) that is introduced in #166 removes everything, including Ubuntu, WSL, and the Virtual Machine Platform.

Uninstalling everything may lead to some headaches, e.g. the instructions for reinstalling do not directly mention re-enabling the Virtual Machine Platform.

However, uninstalling just Ubuntu should be sufficient. Microsoft has an official FAQ on how to uninstall it.
https://learn.microsoft.com/en-us/windows/wsl/faq#how-do-i-uninstall-a-wsl-distribution-

### PR DESCRIPTION
**Pull request by jamesjuett**
This PR fixes #168 by instructing students to only uninstall Ubuntu.

The uninstall and install commands recommended in the Microsoft resources are unlikely to change, so they are directly encoded into the tutorial (rather than a link to an external page - there also isn't an external page I could find that concisely shows the uninstall + install together).

**Verification**
I worked through this process on a spare Windows machine.

1. Start with no WSL installed at all.
2. Install with `wsl --install` from administrator PowerShell.
3. Restart computer as prompted.
4. Open Ubuntu terminal. It prompts to create a user account. Close the terminal without doing that 😈.
5. Open Ubuntu terminal. `whoami` yields `root`.
6. Open PowerShell as administrator. `wsl --unregister Ubuntu`. (Side note, I had an Ubuntu terminal open. It disappeared.)
7. `wsl --install -d Ubuntu` from same PowerShell
8. Output matches the example in this PR, including prompt to create user account. (The PowerShell terminal morphs into bash.)
9. Double check `whoami`. Also check account has sudo privileges. Also check via new Ubuntu terminal. All is good.

### CODE REVIEW

File: docs/setup_wsl.md [hunk 1]
```diff
@@ -92,10 +92,28 @@ $ whoami
 root  # SOMEthiNG IS WRONG
 ```
 
+We recommend you completely reinstall Ubuntu. (Note this will _delete all of the data associated with Ubuntu_. If you've just installed it, that's fine.)
```
Comment (ohjuny) [inline docs/setup_wsl.md:95]: Im unable to comment on line 89 directly, so commenting here.

Thoughts on removing the sentence "We recommend that you [uninstall](https://www.makeuseof.com/uninstall-wsl-windows/) and [reinstall](#install-wsl) WSL to try again." in line 89?

**Review by ohjuny — APPROVED** [review]
LGTM after removing previous sentence.
Also this reinstall method seems much faster and less complicated than the other one, which is great!

File: docs/setup_wsl.md (hunk 1 above)
Comment (awdeorio) [reply docs/setup_wsl.md:95]: +1.  Another option would be to replace the link with the docs suggested by @jamesjuett

**Review by awdeorio — APPROVED** [review]
LGTM, this PR is a good suggestion

File: docs/setup_wsl.md (hunk 1 above)
Comment (jamesjuett) [reply docs/setup_wsl.md:95]: Thanks for pointing this out. I had intended to remove that but missed it! Fixed in 3449ec7

**Pull request merged**